apiclient/request_strategies.py

```diff
@@ -134,24 +134,20 @@ def _decode_response_data(self, response: Response):
 
     def _handle_bad_response(self, response: Response):
         """Convert the error into an understandable client exception."""
-        exception_class = self._get_exception_class(response.status_code)
+        exception_class = self._get_exception_from_client(response)
+        if not exception_class:
+            exception_class = self._get_exception_class(response.status_code)
```

Comment:
I would push to keep this as it was, and remove the exception class from client.

Reply:
👌